src/actions/commands/actions.ts

```diff
@@ -671,23 +672,27 @@ class CommandMoveHalfPageDown extends CommandEditorScroll {
       to: this.to,
       by: this.by,
       value: timesToRepeat,
-      revealCursor: false,
+      revealCursor: smoothScrolling ? true : false,
```

Comment:
Can be simplified to just `smoothScrolling`? No need for ternary operator.

Reply:
:man_facepalming: Good catch, my bad.